airflow/providers/amazon/aws/hooks/glue.py

```diff
@@ -110,24 +110,36 @@ def initialize_job(self, script_arguments=None):
             self.log.error("Failed to run aws glue job, error: %s", general_error)
             raise
 
-    def job_completion(self, job_name=None, run_id=None):
+    def get_job_state(self, job_name=None, run_id=None):
         """
         :param job_name: unique job name per AWS account
         :type job_name: str
         :param run_id: The job-run ID of the predecessor job run
         :type run_id: str
         :return: Status of the Job if succeeded or stopped
```

Comment:
That is not correct here. `get_job_state` can return any state?!

Reply:
Right. It can return any state of actual Glue job can have so I'm updating the description.